project/Dependencies.scala

```diff
@@ -43,7 +43,7 @@ object Dependencies {
     val httpClient  = "0.22.12"
     val logging     = "1.5.1"
     val pubsubFs2   = "0.18.1"
-    val scio        = "0.11.4"
+    val scio        = "0.11.5"
```

Comment:
I'm always nervous when the explicit `beam` version doesn't match the one in the scio library. I usually tend to bump them in a single commit: upgrade scio and simultaneously upgrade beam to match the version in scio. But maybe I'm being unreasonable.

Unrelated to this, apparently the Beam version to use is indeed 2.38.0, because the previous two contain a regression bug. I think it wasn't the case when you opened the PR, but in the meantime there is scio 0.11.6 with beam 2.38.0; and even scio 0.11.7, with bug fixes.

Reply:
As you say, I was forced to use beam 2.38.0 because that was where the vulnerability got fixed.  And I was forced to use scio 0.11.5 because that was the latest scio version when I opened the PR.

Now I will update to 0.11.6.